examples/datahub/main.tf

```diff
@@ -0,0 +1,30 @@
+provider "alicloud" {
+  alias = "bj"
+  region = "cn-beijing"
+}
+
```

Comment:
If there is only one provider, it can be ignored or `alias` is needless.

Reply:
done